scripts/build

```diff
@@ -148,6 +155,8 @@ if [[ ! -d $STEP_DIR ]]; then
     $MAKE install-gcc install-target-libgcc || exit 34
   ) || err $?
 
+  export CXXFLAGS="$ORIGINAL_CXXFLAGS"
```

Comment:
I needed to reset the Darwin-clang specific flags after I was done compiling so it wouldn't interfere with the other compiles. I can move it to the beginning of the next step if you'd like.